app/qml/ui/OptionsDialog.qml

```diff
@@ -208,6 +210,19 @@ Dialog {
                         checked: settings.fpsVisible
                     }
 
+                    Label {
+                        text: qsTr("Always show crosshair cursor")
+                    }
+                    CheckBox {
+                        id: alwaysShowCrosshairCheckBox
+                        leftPadding: 0
+                        checked: settings.alwaysShowCrosshair
+
+                        ToolTip.text: qsTr("Don't hide crosshair cursor when rectangle cursor is visible")
```

Comment:
This is a bit confusing because it seems like it relies on the user knowing implementation details. Can you please make it say:

"Always show the crosshair cursor, even at higher zoom levels and tool sizes"

Reply:
Oh wait, I see what it does now... it actually shows them _both at the same time_. That's fine then. :)